sample-app/build.gradle

```diff
@@ -12,5 +12,12 @@ dependencies {
 	// this logback version is the oldest one available that I could get to work and possibly incompatible
 	// with the one used in the agent. This way, we can test if the shadowing works correctly
 	implementation 'ch.qos.logback:logback-core:1.0.0'
-	implementation 'ch.qos.logback:logback-classic:1.0.0'
+	implementation 'ch.qos.logback:logback-classic:1.0.0'	
+}
+
+// Needed to make git properties work with Java 8, see https://github.com/n0mer/gradle-git-properties/issues/195
+configurations.all {
+	resolutionStrategy { 
+		force 'org.eclipse.jgit:org.eclipse.jgit:5.13.0.202109080827-r'
```

Comment:
This does not have an effect on the buildscript classpath (see failing build). 

Could guess that [this solution](https://github.com/n0mer/gradle-git-properties/issues/195#issuecomment-982326268) is required.